src/solver/hydro/management/management.cpp

```diff
@@ -117,12 +117,10 @@ double BetaVariable(double a, double b, MersenneTwister& random)
 HydroManagement::HydroManagement(const Data::AreaList& areas,
                                  const Data::Parameters& params,
                                  const Date::Calendar& calendar,
-                                 unsigned int maxNbYearsInParallel,
                                  Solver::IResultWriter& resultWriter):
     areas_(areas),
     calendar_(calendar),
     parameters_(params),
-    maxNbYearsInParallel_(maxNbYearsInParallel),
```

Comment:
**maxNbYearsInParallel_** unused, and for good reasons : inside a year, why would we need the number of parallel years ?